docs/core/index.md

```diff
@@ -7,137 +7,41 @@ ms.date: 06/20/2016
 
-- **Flexible deployment:** Can be included in your app or installed side-by-side user- or machine-wide.
-- **Cross-platform:** Runs on Windows, macOS and Linux; can be ported to other operating systems. The [supported Operating Systems (OS)](https://github.com/dotnet/core/blob/master/roadmap.md), CPUs and application scenarios will grow over time, provided by Microsoft, other companies, and individuals.
-- **Command-line tools:**  All product scenarios can be exercised at the command-line. 
-- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin and Mono, via the [.NET Standard](../standard/net-standard.md).
-- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. Documentation is licensed under [CC-BY](https://creativecommons.org/licenses/by/4.0/). .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
-- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://www.microsoft.com/net/core/support/)
+## Download .NET Core 2.1
 
-## Composition
+Download the [.NET Core  2.1 SDK](https://www.microsoft.com/net/download) to try .NET Core on your Windows, macOS or Linux machine. Visit [microsoft/dotnet](https://hub.docker.com/r/microsoft/dotnet/) if you prefer to use Docker containers.
 
-.NET Core is composed of the following parts:
+All .NET Core versions are available at [.NET Core Downloads](https://www.microsoft.com/net/download/archives) if you are looking for another .NET Core version.
 
-- A [.NET runtime](https://github.com/dotnet/coreclr), which provides a type system, assembly loading, a garbage collector, native interop and other basic services. 
-- A set of [framework libraries](https://github.com/dotnet/corefx), which provide primitive data types, app composition types and fundamental utilities. 
-- A [set of SDK tools](https://github.com/dotnet/cli) and language compilers ([Roslyn](https://github.com/dotnet/roslyn) and [F#](https://github.com/microsoft/visualfsharp)) that enable the base developer experience, available in the [.NET Core SDK](sdk.md).
-- The 'dotnet' app host, which is used to launch .NET Core apps. It selects the runtime and hosts the runtime, provides an assembly loading policy and launches the app. The same host is also used to launch SDK tools in much the same way.
+## .NET Core 2.1
 
-### Languages
+The latest version is [.NET Core 2.1](whats-new/index.md). New features include: global tools, high-performance APIs (such as <xref:System.Span%601?displayProperty=nameWithType>), tiered JIT compilation, [build](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1/) and [runtime performance improvements](https://blogs.msdn.microsoft.com/dotnet/2018/04/18/performance-improvements-in-net-core-2-1/), and  support for Alpine and ARM32.
```

Comment:
should we link to the 2.1 what's new here instead?

and also remove extra space between and and support